Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.0.0-dev
+VERSION ?= 0.1.0-beta
```

Comment:
FWIW this was intentionally set to 0.0.0-dev to match our other projects (vault-k8s, csi-provider) and to also have one less place to bump the version at release time, but I get that kustomize is making this awkward.

When developing locally, this might make it harder to tell if the released image or the built image is in use? If we can't keep it at 0.0.0-dev, maybe something else besides the released version? Whatever we choose it'll just have to match the kustomize version I guess.

Reply:
Yeah, I fine with either approach. I think this may have been inadvertently introduced when we did the initial release. There be a way to have `kustomize edit` not munge the file in question.

Reply:
Also fine with either approach, maybe lets stick with precedent from the other projects?

Reply:
I think we should probably set the dev version in the `kustomization.yaml`, then we can avoid the perpetual diff. We can then document the steps needed to install a production version using kustomize.

Reply:
> Also fine with either approach, maybe lets stick with precedent from the other projects?

@kschoche which other projects?

Reply:
Vault-k8s and csi-provider I believe is what Theron had mentioned.

Reply:
Ah, I missed that.

It looks like vault-k8s does not test the kustomize, so it probably doesn't suffer from the same issue. It looks like the vault-k8s kustomize is tied to a release. One thing we could do is to copy the kustomize root to `build/.` before applying it (for the dev case). Then we wouldn't have to an edit in place.